### PR TITLE
[May18] Put things into namespaces and cleaned up the classes

### PR DESCRIPTION
**Pull request by StephenHodgson**
Overview
---
Moved classes into namespaces and misc cleanup & formatting.

Changes
---
- Fixes: #1374

### CODE REVIEW

File: Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs [hunk 1]
```diff
@@ -1,47 +1,55 @@
...
 using UnityEditor;
 using UnityEditor.Callbacks;
+
+#if UNITY_IOS
+using UnityEngine;
 using System.IO;
+#endif // UNITY_IOS
 
-public class UnityARBuildPostprocessor
+namespace ARKit.Utils
```
Comment (davidkline-ms) [inline Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs]: Is this something we are doing for future proofing? UnityARKit came from Unity

Comment (StephenHodgson) [reply Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs]: There wasn't a namespace, and it seems these classes were added outside of the UnityARKit.

Comment (StephenHodgson) [reply Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs]: I just matched it with the folder name.

Comment (keveleigh) [reply Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs]: Looks like this came namespace-less from the plug-in:

https://bitbucket.org/Unity-Technologies/unity-arkit-plugin/src/b4d4bbf5f4bfe454cad3fa2ecd897f5b785952ea/Assets/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs?at=default&fileviewer=file-view-default

File: Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Helpers/AR3DOFCameraManager.cs [hunk 2]
```diff
@@ -1,74 +1,78 @@
-﻿
-using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
 using UnityEngine.XR.iOS;
 
-public class AR3DOFCameraManager : MonoBehaviour {
```
Comment (keveleigh) [inline Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Helpers/AR3DOFCameraManager.cs]: Also came namespace-less:
https://bitbucket.org/Unity-Technologies/unity-arkit-plugin/src/b4d4bbf5f4bfe454cad3fa2ecd897f5b785952ea/Assets/UnityARKitPlugin/Plugins/iOS/UnityARKit/Helpers/AR3DOFCameraManager.cs?at=default&fileviewer=file-view-default

File: Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Helpers/UnityARCameraManager.cs [hunk 3]
```diff
@@ -1,90 +1,107 @@
...
-using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.XR.iOS;
 
-public class UnityARCameraManager : MonoBehaviour {
-
-    public Camera m_camera;
-    private UnityARSessionNativeInterface m_session;
-    private Material savedClearMaterial;
+namespace ARKit.Utils
```
Comment (keveleigh) [inline Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Helpers/UnityARCameraManager.cs]: https://bitbucket.org/Unity-Technologies/unity-arkit-plugin/src/b4d4bbf5f4bfe454cad3fa2ecd897f5b785952ea/Assets/UnityARKitPlugin/Plugins/iOS/UnityARKit/Helpers/UnityARCameraManager.cs?at=default&fileviewer=file-view-default

File: Assets/HoloToolkit-Examples/SpatialSound/Scripts/LoFiFilterSelection.cs [hunk 4]
```diff
@@ -2,100 +2,105 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using UnityEngine;
-using HoloToolkit.Unity;
 using HoloToolkit.Unity.InputModule;
 
-public class LoFiFilterSelection : MonoBehaviour, IInputClickHandler
+namespace HoloToolkit.Unity
```
Comment (keveleigh) [inline Assets/HoloToolkit-Examples/SpatialSound/Scripts/LoFiFilterSelection.cs]: Should this be Examples?

File: Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs (hunk 1 above)
Comment (StephenHodgson) [reply Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs]: Grr. Well not sure what to do then.  I think it's bad practice to go namespaceless.

But to that end, did we take the package as is, or were there other edits?

File: Assets/HoloToolkit-Examples/SpatialSound/Scripts/LoFiFilterSelection.cs (hunk 4 above)
Comment (StephenHodgson) [reply Assets/HoloToolkit-Examples/SpatialSound/Scripts/LoFiFilterSelection.cs]: yes, you're correct.

**Comment by StephenHodgson** [comment]
Kind of maddening that Unity has both GitHub or BitBucket.  Wonder if it's up to the PMs on which to use?

File: Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs (hunk 1 above)
Comment (keveleigh) [reply Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/Editor/UnityARBuildPostprocessor.cs]: I agree about preferably not going namespaceless :/

As I understand, there were other edits to allow building for UWP. Likely, we should try to contribute those (and these namespace edits) back to the original repo.

**Comment by keveleigh** [comment]
@StephenHodgson Seems really strange

**Pull request closed without merging**